docs/docs/general/02-Getting Started/write-your-first-instruction.md

```diff
@@ -51,74 +48,130 @@ Then, go to your `helper/jest.setup.ts` and add the configuration for your `work
-      .exec();
   });
 });
 ```
 
-To execute the automation suite, enter
+To execute the instructions, enter into your terminal
 
 ```shell
 npx jest test/my-first-askui-test-suite.test.ts --config ./test/jest.config.ts 
 ```
 
-You should now see the automation suite being executed inside the shell and the following things happening on your main display:
+A few seconds later an interactive annotation will be generated.
 
-1. **Line 6**: Show an interactively annotated version of your main display with red bounding boxes around the annotated elements. When you press `ESC` on your keyboard the automation will resume.
+If you’d like a longer explanation as to what an interactive annotation is, read about here.
 
-<details>
-  <summary>What is Interactive Annotation?</summary>
-The interactive annotation command requests the askui server to take a screenshot of the specified screen. Then, an AI model is used to annotate the image. After that, a full-screen window appears. Inside this window, boundary boxes enclosing the UI elements detected are going to appear. You can hover over the boxes to see the boxes' specifications, e.g., text detected, element type etc.
-</details>
+[Explanation of Interactive Annotations](../05-Tooling/annotation.md)
+
+### Step 2: Extract the filter
+Extract from the interactive annotation the filter which identifies target element.
+Locate any element you’d like the mouse to move to and copy the filter by clicking on it:
+Clicking an element will copy this filter, which we can then use in the step 3.
+
+To close out the interactive annotation, use `CMD/CTRL + W` or `ESC`.
+
+### Step 3: Execute an Instruction
+
+Add this instruction code block to the describe block in your test file just under your interactive annotation instruction, taking note to also add your copied filter from the annotation:
+
+
+```typescript title="test/my-first-askui-test-suite.test.ts" showLineNumbers
+it('should click on my element', async () => {
+  await aui
+    .click()
+    // <INSERT YOUR COPIED FILTER HERE AND UNCOMMENT THIS LINE>.exec();
+});
+```
+
+Be sure to `xit` out the interactive annotation, as that is no longer needed. The final version should look like this, taking exception of course to whatever filter text you chose:
 
-2. **Line 8**: Your mouse pointer moves to the upper left corner of your main display.
-3. **Line 12**: Your mouse pointer comes back to click on the text `Click on this text right here!` in your automation file.
+```typescript title="test/my-first-askui-test-suite.test.ts" showLineNumbers
+describe('jest with askui', () => {
+
+  xit('should generate an interactive annotation', async () => {
+    await aui.annotateInteractively();
+  });
+
+  it('should click on my element', async () => {
+    await aui
+      .click()
+      .text().withText("node_nodu")
+      .exec();
+  });
+});
+````
+
+As before, run your code with `npx jest --config ./test/jest.config.ts`
```

Comment:
not same code , 
the prev code was `npx jest test/my-first-askui-test-suite.test.ts --config ./test/jest.config.ts `